recipe/meta.yaml

```diff
@@ -30,51 +31,49 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - patch   # [not win]
+    - m2-patch    # [win]
   host:
+    - pip
+    - python
     - cython
+    - docutils
```

Comment:
Yes, it is still required because it is a dependency for `orange-canvas-core`, `recommonmark` and `sphinx`. I thought it wasn't required either, but when I tried removing it before, the builds failed locally.